doc/rst/mason-packages/start/whymason.rst

```diff
@@ -0,0 +1,108 @@
+:title: Mason Guide: *Why mason?*
+        
+Why would I want to use mason?
+==============================
+
+So, you want to write some Chapel code and find yourself asking if it would
+make sense to use or if it would just end up adding needless overhead to your
+project in the long run. Here's an answer for you: Yes! You should use mason!
+
+Using mason brings with it a number of valuable features that are not available
+to users of the old-fashioned ``chpl`` command. Some of these are:
+
+* access to libraries contributed by the Chapel community (see :ref:`using-a-package`)
+  
+* portable, easy build commands (see :ref:`building-and-running`)
+  
+* an existing testing framework to help test your code (see :ref:`testing-with-mason`)
+  
+* make your code easily shareable on GitHub (see :ref:`mason-git-dependencies`)
+  
+* contribute your own libraries to the existing mason registry (see :ref:`submit-a-package`)
+  
+
+So, you see, a better question would be "Why not use mason?" instead of "Why use mason?"
```

Comment:
> So, you see, a better question would be "Why not use mason?" instead of "Why use mason?"

Why use mason? In short, mason makes it easier to:

* build and run your code
* use Chapel libraries in your code
* test your code
* share your code
* include example usages of your code